packages/demo/src/components/Bandada/Bandada.tsx

```diff
@@ -1,24 +1,58 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+
+import { ActionBox } from "../ActionBox/ActionBox";
+
 interface IBandadaProps {
   joinGroup: () => Promise<void>;
   generateGroupMerkleProof: () => Promise<void>;
 }
 
+const GENERATE_MERKLE_TREE_CODE = `import { initializeCryptKeeper } from "@cryptkeeperzk/providers";
+
+const client = initializeCryptKeeper();
+
+const connect = (async () => {
+  await client?.request({
+    method: RPCExternalAction.GENERATE_GROUP_MERKLE_PROOF,
+    payload: {
+      groupId: process.env.TEST_GROUP_ID!,
+    },
+  });`;
+
+const JOIN_GROUP_CODE = `import { initializeCryptKeeper } from "@cryptkeeperzk/providers";
+
+const client = initializeCryptKeeper();
+
+const connect = async () => {
+  await client?.request({
+    method: RPCExternalAction.JOIN_GROUP,
+    payload: {
+      groupId: process.env.TEST_GROUP_ID!,
+      apiKey: process.env.TEST_GROUP_API_KEY,
+      inviteCode: process.env.TEST_GROUP_INVITE_CODE,
+    },
+  })`;
```

Comment:
Can we move it to text files?

Reply:
Why we need to do that? I think its fine to have it like a string `const` in each component.

Reply:
It's like a data format and it will be easier to edit it and read.
It can be a separate file next to component.

Reply:
The thing is I dont want to make it complicated for now, that would require to use `fs` to access file system. 
I tried also using fetch: 
```
export const loadFile = async (filePath: string): Promise<string> => {
  try {
    const response = await fetch(filePath);
    const data: string = await response.text();
    return data;
  } catch (error) {
    console.error("Error loading the file:", error);
    throw error;
  }
};
```

And that is the example: 
```
export const GetMetadata = ({ getConnectedIdentityMetadata }: IGetMetadataProps): JSX.Element => {
  const classes = useGlobalStyles();

  const [codeExample, setCodeExample] = useState("");

  const getCodeExample = useCallback(async () => {
    const codeExampleResponse = await loadFile("./codeExample.ts");
    setCodeExample(codeExampleResponse);
  }, [setCodeExample])

  useEffect(() => {
    getCodeExample();
  }, [getCodeExample]);

  return (
    <Box
      className={classes.popup}
      sx={{ p: 3, display: "flex", flexDirection: "column", alignItems: "center", justifyContent: "center" }}
    >
      <Typography variant="h6">Get Connected Identity Metadata</Typography>

      <ActionBox<undefined, void>
        code={codeExample}
        option={undefined}
        title="Get Connected Identity"
        onClick={getConnectedIdentityMetadata}
      />
    </Box>
  );
};
```

And that is the output: 
![Screenshot_20231023_122429](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/36774944/e776f413-6860-44e4-8d61-c64ad7997689)


I think this issue could be resolved in a better way in future enhancements.

Reply:
What do you think of this idea? 
1. Having a `codeExample.ts` script on each component like this: https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/feature/demo-styles/packages/demo/src/components/Bandada/codeExample.ts
2. Having a `codeExported.ts` script on each component like this: https://github.com/CryptKeeperZK/crypt-keeper-extension/blob/feature/demo-styles/packages/demo/src/components/Bandada/codeExported.ts

The features of this approach: 
1. Simple. 
2. Making sure that the code snippet is correct `ts` code, and has got eslint checks. 
3. Copy the code from `codeExample.ts` and then paste it in `codeExported.ts` variables.

Reply:
I think with fetch is a good one with `codeExample.ts`. So it can be copy paste by dev right from demo folder.

Reply:
I had to use a script for copying the code examples into the dist of the demo and then would be able to fetch them.